pymeasure/instruments/chroma/chroma_66205.py

```
@@ -0,0 +1,470 @@
+        "TRIGGER?",
+        "TRIGGER %s",
+        """Set trigger for energy calculation, inrush, or limit (go/no-go).
+        Status is STOP|FINISH|RUNNING.
+        To trigger, pass 'ON'. To stop or reset integration cycle, pass 'OFF'.""",
+    )
+
+    def measure(self,param: Measure):
+        """Get a measurement. See Chroma66205.Measure."""
+        return float(self.ask(f"MEAS? {param.value}").strip())
+
+    def integrate_energy(self,integration_time_s: float=10.):
```

Flake8 (7.3.0)

[error] missing whitespace around parameter equals

(E252)